cross/x86_ci_script.sh

```diff
@@ -1,64 +0,0 @@
-#!/bin/bash
```

Comment:
Was the deletion of this script intentional? It doesn't seem to exist in the eng/common/cross folder.

Reply:
yes, I realized that. I assumed all needed was moved over but it was not. I'm trying to find out how to undelete it. I may abandon this PR and create new and  remove only files related to build-rootfs